a3fe/run/system_prep.py

```diff
@@ -184,6 +190,29 @@ class Config:
         extra = "forbid"
         validate_assignment = True
 
+    def get_membrane_equilibration_config(self) -> dict:
+        """Get the membrane configuration for equilibration runs."""
+        if self.membrane_protein:
+            return {
```

Comment:
Are all of these options different to the default? I'm assuming that they are, but if not it might be good to only include the options which are different to default, so that it's clear what changes when you set `self.membrane_protein = True`. Minor comment though - feel free to ignore.